web-registry/src/utils/number/format.ts

```diff
@@ -0,0 +1,12 @@
+type RoundFloatNumberParams = {
+  number: number;
+  decimals?: number;
+};
+
+export const roundFloatNumber = ({
+  number,
+  decimals = 2,
+}: RoundFloatNumberParams): number => {
```

Comment:
Can't remember where I mentioned something similar in another story, but minor nit: I'd prefer having `number` as the first argument and `options` as the second
```tsx
export const roundFloatNumber = (
  num: number,
  options?: { decimals?: number },
): number => {
  const decimals = options?.decimals ?? 2
  const base = Math.pow(10, decimals)
  return Math.round(num * base) / base
}
```

it's a pretty simple helper function, so having to explicitly pass `roundFloatNumber({ number: 1.234 })`  is a bit tedious compared to `roundFloatNumber(1.234)`

Reply:
@haveanicedavid  Fair point but I have a counter-argument to that, function without named params tends to be harder to extend. I had a case several times recently where I had to change the signature of functions without named params and I had to update the signature everywhere, which was a bit painful. So even if it seems shorter to write `roundFloatNumber(1.234)` now I would rather have `roundFloatNumber({ number: 1.234 })` to make it more easily extendable in the future.

Reply:
Well after second thinking I think for this case the option solution might be good enough. In anycase it would be probably best to use named parameter by default in favor of extendabilty and use that solution only when it makes sense.

Reply:
My take is that in this case, the functions purpose is fairly single-minded, and since it doesn't seem like a large or complex set of dependencies in the function signature, it makes sense to me to have a simple call signature, with options, i.e. `roundFloatNumber(1.234, {decimals: 10})`